config/webrtc.gclient.common

```diff
@@ -7,7 +7,7 @@
 
 solutions = [
   { "name"        : "src",
-    "url"         : "git@github.com:signalapp/webrtc.git",
+    "url"         : "https://github.com/gpotskhveria/webrtc.git",
```

Comment:
We have an update for this on the way, and referring to signalapp/webrtc.git over https.